packages/libs/client/src/composePactCommand/utils/patchCommand.ts

```diff
@@ -47,6 +50,16 @@ export const mergePayload = (
   throw new Error('PAYLOAD_NOT_MERGEABLE');
 };
 
+/**
+ * Merge a partial command on top of the command
+ *
+ * @remarks
```

Comment:
Nitpick: unknown JSDoc tag?

Reply:
It's TSDoc. Amongst others, JSDoc includes types which are redundant in TSDoc